processor/schemaprocessor/internal/translation/revision_v1.go

```diff
@@ -6,109 +6,144 @@ package translation // import "github.com/open-telemetry/opentelemetry-collector
 import (
 	"go.opentelemetry.io/otel/schema/v1.0/ast"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/schemaprocessor/internal/changelist"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/schemaprocessor/internal/migrate"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/schemaprocessor/internal/transformer"
 )
 
 // RevisionV1 represents all changes that are to be
 // applied to a signal at a given version.
+// todo(ankit) implement split and rest of otel schema
```

Comment:
It would be useful to add a note that says which exactly version of Schema Files this implements. Looks like it implements 1.0, but not 1.1.

Also would be useful to somehow make clear that `V1` in the name here refers to the Schema File format version, not schema version.

Reply:
goood idea!